### PR TITLE
Add some PD client metrics

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: src/pd/pd.rs [hunk 1]
```diff
@@ -382,6 +349,8 @@ impl<T: PdClient> Runner<T> {
                     );
                     let req = new_transfer_leader_request(transfer_leader.take_peer());
                     send_admin_request(&ch, region_id, epoch, peer, req, None)
+                } else {
+                    PD_HEARTBEAT_COUNTER_VEC.with_label_values(&["noop"]).inc();
```
Comment (siddontang) [inline src/pd/pd.rs]: what does noop mean here?

Comment (huachaohuang) [reply src/pd/pd.rs]: It means that we receive a response but we don't need to do anything.

**Comment by siddontang** [comment]
PTAL @disksing

File: src/pd/metrics.rs [hunk 2]
```diff
@@ -14,24 +14,17 @@
 use prometheus::*;
 
 lazy_static! {
-    pub static ref PD_SEND_MSG_HISTOGRAM: Histogram =
-        register_histogram!(
-            "tikv_pd_msg_send_duration_seconds",
-            "Bucketed histogram of PD message send duration",
-             exponential_buckets(0.0005, 10.0, 7).unwrap()
```
Comment (disksing) [inline src/pd/metrics.rs]: Why not keep original buckets?

Comment (huachaohuang) [reply src/pd/metrics.rs]: It seems not very useful, and can't distinguish the message type.

**Comment by disksing** [comment]
LGTM.

**Pull request merged**